hdd_tools/config.json

```diff
@@ -38,6 +39,7 @@
       "database_update_period": "int(1,5000)",
       "performance_check": "bool",
       "debug": "bool",
-      "output_file": "str"
+      "output_file": "str",
+      "device_type": "str"
```

Comment:
Move this too after `hdd_path`.

The easier way for users is to make this a list with all the possible options, but I have seen different documentation, some of them explain 10 or 20 possible values, that is ok, but others comment more than 200 because you can select the device number... do you know how many different options exist?

Reply:
I don't think that maintaining all the possible options is reasonable. Just let the user read the documentation and place a value.

Reply:
Ok. Seems reasonable to me.